src/devices/Hts221/Hts221.csproj

```diff
@@ -12,6 +12,7 @@
       <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
     </ProjectReference>
     <ProjectReference Include="..\Units\Units.csproj" />
+    <ProjectReference Include="..\Common\Iot\Device\Common\CommonHelpers.csproj" />
```

Comment:
Is this actually using CommonHelpers or is only sample project using it?

Reply:
The sample project, but it might use it in future. Is this a problem?

Reply:
Done (need to commit)